lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
Ask and you shall receive (should have known and looked though). If @trotzig agrees, I can buy moving that way.

Note that changing ESLint right now to have 80 char comments and 100 char code limits would probably reject a lot of comments I've made. If we all agree to go that way, I need to make some changes before it is turned on.